pkg/api/serialization_test.go

```diff
@@ -57,6 +57,9 @@ func fuzzInternalObject(t *testing.T, forVersion string, item runtime.Object, se
 func roundTrip(t *testing.T, codec runtime.Codec, item runtime.Object) {
 	printer := spew.ConfigState{DisableMethods: true}
 
+	gvk, err := api.Scheme.ObjectKind(item)
+	t.Logf("fqKind for %v is %v with codec %v", reflect.TypeOf(item), gvk, codec)
```

Comment:
> not sure if fqKind is used elsewhere, but prefer "fully qualified kind"

As the message or as the variable name?  `fullyQualifiedKind` seems a bit long, I was hoping for `fqKind` as a compromise